modular_skyrat/modules/hyposprays/code/hypovials.dm

```diff
@@ -46,31 +46,31 @@
 /obj/item/reagent_containers/cup/vial/small
 	name = "hypovial"
 	desc = "A small, 60u capacity vial compatible with most hyposprays."
```

Comment:
Hello! I noticed both this desc will no longer will match the actual volume. Please remember to update it to reflect the new volume.

Reply:
Oop, yeah